fastly.toml

```
@@ -1,8 +1,15 @@
-manifest_version = 2
-name = "Static content"
-description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3."
+# This file describes a Fastly Compute@Edge package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
 authors = ["<oss@fastly.com>"]
+description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3."
 language = "rust"
+manifest_version = 2
+name = "Static content"
+service_id = ""
+
+[scripts]
+  build = "cargo build --bin fastly-compute-project --release --target wasm32-wasi --color always"
```

These changes are unnecessary.

```suggestion
manifest_version = 2
name = "Static content"
description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3."
```

---

I agree that these changes have zero effect, but they also happen immediately on running `fastly compute serve` for example, which might leave some users confused.

---

Actually there was a reason for not adding them and that was for older versions of the CLI to not break 😬 

So historically, the CLI contained all the logic for building a project and `[scripts.build]` was used as an _optional_ override. If you set it, then "all bets were off" as far as validation was concerned (as the CLI would no longer attempt to validate your local environment).

The CLI was then updated so that `[scripts.build]` was _mandatory_ and it was the CLI would use it as the defacto way to know how to build a project. The intention was (as part of that CLI release) to update all starter kits with relevant build scripts BUT we then realised there was an issue with doing that...

Users of older versions of the CLI will see `[scripts.build]` as an 'override' and so when pulling down a starter-kit with a build script predefined it would mean they get no validation from the CLI and even a warning message telling them that there was a build script found in their fastly.toml and to be careful in case it was malicious.  

The intention was to, at some future point, when Sentry tells us there are no more users with CLI version `<4.0.0` (I think it was that release we changed to a mandatory `[scripts.build]`) then we'd update all the starter kits to include a build script (that way the newer CLI versions wouldn't have to inject their own -- which was a fallback to help support both older and newer CLI versions).

Yup. Things got complicated 😬